elixir/LICENSE

```diff
@@ -0,0 +1,18 @@
+Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
... then at least have some "MIT License" at the top so people know what this is